Makefile

```diff
@@ -37,7 +37,7 @@ endif
 OCCA_DIR ?= ../occa
 
 # env variable MAGMA_DIR can be used too
-MAGMA_DIR ?= ../magma
+MAGMA_DIR ?= ../../magma
```

Comment:
Is this change important?

Reply:
In what sense is this "resolved"? This pattern is different from every other package here. If you want to change it for everyone, there should be a reason.